src/psyclone/tests/gocean1p0_transformations_test.py

```diff
@@ -1184,3 +1184,98 @@ def test_module_inline_warning_no_change():
     kern_call = schedule.children[0].children[0].children[0]
     inline_trans = KernelModuleInlineTrans()
     _, _ = inline_trans.apply(kern_call, inline=False)
+
+
+def test_loop_swap_correct():
+    ''' Testing correct loop swapping transform. Esp. try first, middle, and
+    last invokes to make sure the inserting of the inner loop happens at
+    the right place.'''
+
+    psy, invoke = get_invoke("test27_loop_swap.f90", 0)
+    invoke = psy.invokes.get("invoke_loop1")
+    schedule = invoke.schedule
+    schedule_str = str(schedule)
+
+    # First make sure to throw an early error if the source file
+    # test27_loop_swap.f90 should have been changed
+    expected_schedule = '''Loop[]: j= lower=2,jstop,1
+Loop[]: i= lower=2,istop,1
+kern call: bc_ssh_code'''
+    assert expected_schedule in  schedule_str
+
+    expected_schedule = '''Loop[]: j= lower=1,jstop+1,1
+Loop[]: i= lower=1,istop,1
+kern call: bc_solid_u_code'''
+    #assert expected_schedule in  schedule_str
```

Comment:
Please remove commented-out assert. In fact, should this be commented out as the value just assigned to `expected_schedule` is not used before that variable is assigned a different value?